src/main/java/com/rcv/ResultsWriter.java

```diff
@@ -681,7 +681,7 @@ void generateCdfJson(List<CastVoteRecord> castVoteRecords)
     Map<String, Object> contestMap =
         Map.ofEntries(
             entry("ContestId", CDF_CONTEST_ID),
-            entry("CVRContestSelection", selectionMapList),
+            entry("ContestSelection", selectionMapList),
```

Comment:
I couldn't tell for sure from John's email whether this was the thing he wanted us to change.

Reply:
Yeah, his email (and the standard) are confusing, but I think it's right.  Looking at all the other @type elements it checks out.